script/sanitizers.supp

```diff
@@ -0,0 +1,4 @@
+[undefined]
+# This library allows unaligned access on Intel-like processors. Prevent UBSan
+# from complaining about that.
+fun:sha1_compression_states
```

Comment:
This probably belongs into #5569, right?

Reply:
Done.